### PR TITLE
[Fix] mask function and vfmv head file name

### PR DESCRIPTION
**Pull request by XYenChi**
1.  Initiate vd default data for mask intrinsic function and put it at the end of declaration.
2.  Fix vfmv header file name for missing _f or _s.

### CODE REVIEW

File: scripts/VXLiteral.py [hunk 1]
```diff
@@ -442,6 +443,8 @@ def create_vx_op(op_type, op_id, op_attr, output_type, input_num, input_types) :
   var = chr(ord('a') + input_num)
   ret += "  auto " + var + " = static_cast<RIF::" + output_type + "Val *>(op->outputs[0]);\n"
   if "MaskedOperation" in op_attr :
+    var = chr(ord('a') + input_num + 1)
+    ret += "  auto " + var + " = static_cast<RIF::" + output_type + "Val *>(op->inputs[" + str(input_num) + "]); // scripts/VVLiteral.py create_vv_op \n"
```
Comment (kito-cheng) [inline scripts/VXLiteral.py:447]: Drop debug info
```suggestion
    ret += "  auto " + var + " = static_cast<RIF::" + output_type + "Val *>(op->inputs[" + str(input_num) + "]);\n"
```

File: scripts/VXLiteral.py [hunk 2]
```diff
@@ -144,16 +144,17 @@
 '''
 
 vx_literal_mask_body = '''
+  // script/VXLiteral.py vx_literal_mask_body\n
```
Comment (kito-cheng) [inline scripts/VXLiteral.py:147]: Drop debug info

File: scripts/XLiteral.py [hunk 3]
```diff
@@ -166,11 +166,13 @@ def include_literal(filename):
 def create_x_op(op_type, op_id, op_attr, output_type, input_num, input_types) :
   ret = ""
   ret += x_literal_start0 + op_type + x_literal_start1
+  if op_id == "fmv_f":
+    op_id = op_id + "_s"
   for i in range(input_num) :
     var = chr(ord('a') + i)
-    ret += "  auto " + var + " = static_cast<RIF::" + input_types[i] + "Val *>(op->inputs[" + str(i) + "]);\n"
+    ret += "  auto " + var + " = static_cast<RIF::" + input_types[i] + "Val *>(op->inputs[" + str(i) + "]); // scripts/XLiteral.py create_x_op \n"
```
Comment (kito-cheng) [inline scripts/XLiteral.py:173]: Drop debug info

File: scripts/XLiteral.py [hunk 4]
```diff
@@ -166,11 +166,13 @@ def include_literal(filename):
...
   var = chr(ord('a') + input_num)
-  ret += "  auto " + var + " = static_cast<RIF::" + output_type + "Val *>(op->outputs[0]);\n"
+  ret += "  auto " + var + " = static_cast<RIF::" + output_type + "Val *>(op->outputs[0]); // scripts/XLiteral.py create_x_op \n"
```
Comment (kito-cheng) [inline scripts/XLiteral.py:175]: Drop debug info

**Comment by XYenChi** [comment]
Recently I found where the CustomOperator.def was generated should resolve this vfmv headfile name issue.The way I added the condition might cause variable type issue, so I will close this pull request XD

**Pull request closed without merging**